data/mods/National_Guard_Camp/item_groups.json

```diff
@@ -56,13 +56,15 @@
     "id": "weapon_testing_lab",
     "type": "item_group",
     "items": [
-      { "group": "tools_gunsmith", "prob": 60 },
-      { "group": "gunmod_common", "prob": 10 },
-      { "group": "gunmod_milspec", "prob": 10 },
-      { "group": "gunmod_rare", "prob": 10 },
-      { "group": "guns_milspec", "prob": 10 },
-      { "group": "mags_milspec", "prob": 10 },
-      { "group": "mil_armor", "prob": 20 }
+      { "group": "tools_gunsmith", "prob": 65 },
+      { "group": "gunmod_common", "prob": 20 },
+      { "group": "gunmod_milspec", "prob": 20 },
+      { "group": "gunmod_rare", "prob": 17 },
+      { "group": "guns_milspec", "prob": 20 },
+      { "group": "mags_milspec", "prob": 20 },
+      { "group": "energy_weapon_armory", "prob": 17 },
+      { "group": "hightec_guns", "prob": 3 },
```

Comment:
I would recommend changing this to "guns_energy" itemgroup.

Reply:
The energy_weapon_armory group is there because it has laser weapon mods in it. But sure, I'll just add those to the hightec group.